---
layout: default
---
# GB0228

## Graph-Break Type
*Short name describing what triggered the graph break*

`torch.nn.Parameter`: cannot convert to traceable tracable

## Context
*Values or code snippet captured at the break point*



## Explanation
*Explanation of why the graph break was triggered*

convert_tracable_parameter is set to False.

## Hints
*Hints on how to resolve the graph break*

- Check usage of context manager: do_not_convert_to_tracable_parameter
- This graph break may be difficult to debug. Please report an issue to PyTorch for assistance.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->

[Back to Registry](../index.html)

[Edit this page on GitHub](https://github.com/pytorch-labs/compile-graph-break-site/edit/main/docs/gb/gb0228.md)
